src/ngraph/runtime/cpu/pass/cpu_assignment.cpp

```diff
@@ -483,6 +484,30 @@ namespace ngraph
                     }
                 }
 
+                template <>
+                void CPUAssignment::ASSIGN_DECL(ngraph::op::ReplaceSlice)
+                {
+                    auto replace_slice = static_cast<op::ReplaceSlice*>(node);
+
+                    //auto arg0_shape = node->get_input_shape(0);
```

Comment:
remove dead code

Reply:
will do. just wanted to confirm we don't need to use ranks anywhere.
Again, i feel we should just make sure the ranks are same for arg0 and arg1